### DISCUSSION TITLE
What element is also known as quicksilver?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1158] [Galaxy Trivia]
**Bonus:** Which cell phone company made the first commercially successful car radio?

**Comment by tomerf** [comment]
Mercury is the chemical element also known as quicksilver.

The first car radio was developed by the American company Motorola. In 1930, Motorola introduced the Motorola Model 5T71, which was designed to be installed in automobiles.

**Comment by Camarota-234** [comment]
Mercury is also known as quicksilver. 
Fun fact: DC has a speedster superhero who is called Mercury and Marvel has one called Quicksilver.

The first commercially successful car radio was made by Motorola in 1930 and cost about a quarter of the price of a new car. The name Motorola was the junction of "motor" referring to the car and "ola" implying sound (as in a record-playing Victrola).

**Reply by ParanoidUser** [reply]
Thank you for sharing interesting facts! ❤️
